webapp/static/interop-data.json

```diff
@@ -446,44 +445,44 @@
             },
             "interop-2023-mathfunctions": {
                 "description": "Math Functions",
-                "mdn": "",
-                "spec": "",
-                "tests": "/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-mathfunctions", 
+                "mdn": "https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Functions#math_functions",
+                "spec": "https://w3c.github.io/csswg-drafts/css-values-4/#math",
+                "tests": "/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-mathfunctions",
                 "countsTowardScore": true
             },
             "interop-2023-mediaqueries": {
                 "description": "Media Queries 4",
-                "mdn": "",
-                "spec": "",
-                "tests": "/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-mediaqueries", 
+                "mdn": "https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries",
+                "spec": "https://www.w3.org/TR/mediaqueries-4/",
+                "tests": "/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-mediaqueries",
                 "countsTowardScore": true
             },
             "interop-2023-modules": {
                 "description": "Modules",
-                "mdn": "",
-                "spec": "",
-                "tests": "/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-modules", 
+                "mdn": "https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Using_web_workers",
+                "spec": "https://html.spec.whatwg.org/multipage/workers.html#module-worker-example",
```

Comment:
The majority of this focus area is import assertions, so i wonder if https://tc39.es/proposal-import-assertions/ is better?

Reply:
sgtm